doc/COMPILE.md

```diff
@@ -135,23 +135,16 @@ For a list of compilers we test with have a look at:
   on [our build server](https://build.libelektra.org/)
 - [Travis](/.travis.yml)
 - [Cirrus](/.cirrus.yml)
+- [Github Actions](/.github/workflows)
 
 Here is a list of compilers used by developers or build servers:
 
-| Compiler | Version                     | Target              |
-| -------- | --------------------------- | ------------------- |
-| gcc      | gcc (Debian 6.3.0-18) 6.3.0 | amd64               |
-| gcc      | gcc 4.8                     | amd64               |
-| gcc      | gcc 4.9                     | amd64               |
-| clang    | 3.8                         | x86_64-pc-linux-gnu |
-| clang    | 5.0                         | x86_64-pc-linux-gnu |
-| clang    | 6.0                         | x86_64-pc-linux-gnu |
-| clang    | 8.0                         | macOS               |
-| gcc      | 9.1                         | macOS               |
-| gcc/g++  | 4.9.4 (¹)                   | openbsd 6.3         |
-| mingw    | 6.0.0-3                     | amd64               |
-| clang    | 6.0.0                       | freebsd 11          |
```

Comment:
Did we drop all our openbsd/freebsd buildjobs?

Isn't there any job anymore with older compilers?

Reply:
I pushed to the wrong branch here but regardless: 

The compilers from our CI can be seen at any time and the document links to the Dockerfiles, Jenkins etc. (thus removal of FreeBSD ..)

In #3384 we discussed that this list should only contain the manually tested setups by developers. I re-started the list and I hope others will add to it. I do fear that it will get outdated again to be honest. To my knowledge none of us are currently testing neither on OpenBSD nor on ancient GCC versions.

Reply:
Ok, then the introducing sentence is confusing, I made a suggestion.

Yeah, it will easily get outdated again. Probably we should remove this manually-written list in the long term and instead simply refer to Dockerfiles etc. instead.

I already suggested to @robaerd that it would be great if the build jobs would do a `cc --version` in the beginning so that the version info always gets printed and is easily discoverable.